src/main/java/org/folio/circulation/domain/policy/LoanPolicy.java

```diff
@@ -244,8 +248,13 @@ private DueDateStrategy determineStrategy(boolean isRenewal, DateTime systemDate
           getRenewalDueDateLimitSchedules(), this::errorForPolicy);
       }
       else {
-        return new RollingCheckOutDueDateStrategy(getId(), getName(),
-          getPeriod(loansPolicy), fixedDueDateSchedules, this::errorForPolicy);
+        FixedDueDateSchedules appliedDueDateSchedules = Objects.nonNull(alternateRenewalFixedDueDateSchedules)
+          && Objects.nonNull(requestQueue)
```

Comment:
I think we may want to introduce an none Request Queue to handle circumstances like this.

Reply:
WOuld you want to do this wirthin the scope of this issue?

Reply:
No, I'm just planting seeds (partly in my mind) for improvements